### PR TITLE
[10.7] [PR 4500] Move branch specific attributes from global site.yml to branch relevant antora.yml

### PR DESCRIPTION
**Pull request by mmattel**
Backport of PR #4500

### CODE REVIEW

File: antora.yml [hunk 1]
```diff
@@ -8,3 +8,9 @@ nav:
 
 asciidoc:
   attributes:
+    minimum-php-version: 7.3
```
Comment (phil-davis) [inline antora.yml:11]: 10.7 supports PHP 7.2 - I guess you will do a separate PR to update the specific values?

**Review by phil-davis — APPROVED** [review]

File: antora.yml (hunk 1 above)
Comment (phil-davis) [reply antora.yml:11]: Yes, will be done in a next PR

Comment (mmattel) [reply antora.yml:11]: yes, see my question in the 10.8 backport
